v2/argsparser.go

```diff
@@ -222,9 +238,15 @@ parseLoop:
 			case r == '/':
 				incomplete = true
 				st = stTrailingSlash
+			case r == '=':
+				st = stTrailingValue
 			default:
 				break parseLoop
 			}
+		case stTrailingValue:
+			valueStart = i
+			i = len(runes) - 1
+			break parseLoop
```

Comment:
@thockin The idea here was to track where the "beginning of the key value" is found so I can return the position of it in the parse result.

Reply:
Alternate approach, accumulate all bytes, and when you hit the end of the loop `if st == stValue` then save that and return it.  Downside: any trailing comment is still there.  If someone calls this func again, it will "just work"

Reply:
Agreed.  I'll leave the trailing comment in the value and parse it as opaque text.  See below comments for more detail.